sirepo/http_request.py

```diff
@@ -32,10 +32,7 @@ def parse_json(qcall):
     # certain clients have been using this in the past.  This
     # fits our general approach of being nice in what we accept
     # and strict in what we send out.
-    return simulation_db.json_load(
-        qcall.sreq.body_as_bytes(),
-        encoding=qcall.sreq.content_type_encoding(),
-    )
+    return simulation_db.json_load(qcall.sreq.body_as_bytes())
```

Comment:
Once we commit this, do we break f32?

Reply:
This change won't. But, the changes to warppba will (see failing tests now). I guess this can't be merged until we are ready to go on f36

Reply:
https://github.com/radiasoft/sirepo/pull/5223/commits/089800f797606ac25a86f4126dde1871ecec2efb